.eslintrc

```diff
@@ -48,6 +48,10 @@
     "no-throw-literal": 2,
     "no-trailing-spaces": 2,
     "no-unused-expressions": 0,
+    "no-unused-vars": [2, {
+      "argsIgnorePattern": "^var_|opt_|unused_|_",
```

Comment:
The leading _ in closure compiler defaults to "Do not obfuscate".

Any chance you could change the PR to actually use unused? If you did this manually I don't want you to redo it, though.